keyboards/whtkbrd/config.h

```diff
@@ -0,0 +1,55 @@
+/* Copyright 2022 Alexander Metz
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#pragma once
+
+#include "config_common.h"
+
+#define VENDOR_ID 0xFEED
+#define PRODUCT_ID 0x0001
+#define DEVICE_VER 0x0001
+
+#define MANUFACTURER whtk
+#define PRODUCT whtkbrd
+
+#define SPLIT_USB_DETECT
+#define MASTER_RIGHT
+
+#define SERIAL_USART_FULL_DUPLEX
+#define SERIAL_USART_TX_PIN A9
+#define SERIAL_USART_RX_PIN A10
+/* Swap TX and RX pins if keyboard is master halve. Disable this for boards with hardwired crossover. */
+#define SERIAL_USART_PIN_SWAP
+#define SELECT_SOFT_SERIAL_SPEED 1
+#define SERIAL_USART_DRIVER SD1
+#define SERIAL_USART_TX_PAL_MODE 1
+#define SERIAL_USART_RX_PAL_MODE 1
+#define SERIAL_USART_TIMEOUT 20
+
+/* Rows from each half are stacked. */
+#define MATRIX_ROWS 10
+#define MATRIX_COLS 6
+#define DIODE_DIRECTION COL2ROW
+/* Strong debounce as kailh choc keys are bouncy. At least the pro-reds I use. */
+#define DEBOUNCE 50
```

Comment:
This may be better to put into a user keymap, especially with this high of a debounce.

Reply:
Moved to the keymap `config.h`